bigbluebutton-html5/imports/ui/components/lock-viewers/service.js

```diff
@@ -0,0 +1,14 @@
+import { makeCall } from '/imports/ui/services/api';
+import Meetings from '/imports/api/meetings';
+import Auth from '/imports/ui/services/auth';
+
+const getCurrentMeeting = () => Meetings.findOne({ meetingId: Auth.meetingID });
```

Comment:
Why do we need to pass the entire Meeting object? Why not just the meetingId?

Reply:
I just moved these functions to service, but for some unknown reason the method uses the entire meeting.